server/service/src/processors/transfer/requisition/test.rs

```diff
@@ -262,7 +262,7 @@ impl RequisitionTransferTester {
             r.suggested_quantity = 3.0;
             r.comment = Some("line comment".to_string());
             r.available_stock_on_hand = 1.0;
-            r.average_monthly_consumption = 1.0;
+            r.average_monthly_consumption = 0.9999999999999934;
```

Comment:
Example of a rounding change. Maybe to do with plugins work? Wouldn't have expected anything to change there?

Reply:
I think we can revert this, if we pull from 2.6.1-RC branch